README.md

```diff
@@ -143,3 +143,24 @@ func main() {
 
 The main examples have been moved - you can find them in their [own repository](https://github.com/fyne-io/examples/).
 
+## Dependencies
+
+Fyne requires the following imports which are not included in the go standard library:
+
+* github.com/go-gl/gl/v3.2-core/gl
+* github.com/go-gl/glfw/v3.2/glfw
+* github.com/golang/freetype
+* github.com/srwiley/oksvg
+* github.com/srwiley/rasterx
+* golang.org/x/image/font
+
+These can be included in your gopath by the following 'get' commands:
+
+* "go get github.com/go-gl/gl/v3.2-core/gl"
+* "go get github.com/go-gl/glfw/v3.2/glfw"
+* "go get github.com/golang/freetype"
+* "go get github.com/srwiley/oksvg"
+* "go get github.com/srwiley/rasterx"
+* "go get golang.org/x/image/font"
+
+You will also require a GCC compiler if you do not have one, if on Windows this can be included by installing TDM-GCC.
```

Comment:
I agree this GCC mention is probably needed, but can we structure it a little more to be about CGo is required and link to some document about setting this up. If I was new to developing with Windows I'm not sure that "instal TDM-GCC" would be enough detail for me to get up and running?

Reply:
I did some digging and the official Golang documentation states that for Windows C support to use minGW; however many people complain this solution doesn't work and to use TDM-GCC.  Hence I continued to incude it.

What exactly would be required for the CGo details? This shouldn't be a problem if users do a full Go install, usually issues witch CGo arise when a GCC compiler doesn't exist on the system.